projects/ad9083/src/app_iio.c

```diff
@@ -0,0 +1,139 @@
+/***************************************************************************//**
+ *   @file   app_iio.c
```

Comment:
you can use iio_app from noos/iio/iio_app

Reply:
Will do that